packages/preview-middleware/templates/test/qunit.js

```diff
@@ -1,6 +1,8 @@
 sap.ui.define([
     'sap/ui/thirdparty/qunit-2'
 ], function() {
```

Comment:
I'd suggest to rather access `QUnit` from the sap.ui.define import, not via global.

Reply:
oh, you are right 🙈

Reply:
Looks better now 👍🏻